File: gui/hello/blank.py
#! /usr/bin/python

import pygtk
import gtk

class Blank:
    def __init__(self):
        self.window = gtk.Window(gtk.WINDOW_TOPLEVEL)
        self.window.show()

    def main(self):
        gtk.main()

if __name__ == '__main__':
    app = Blank()
    app.main()

File: gui/hello/blank.c
#include <gtk/gtk.h>

int main(int argc, char* argv[])
{
    GtkWidget *window;

    gtk_init(&argc, &argv);     // initialize gtk toolkit

    // build widget tree
    window = gtk_window_new(GTK_WINDOW_TOPLEVEL);
    gtk_window_set_title(GTK_WINDOW(window), "Blank");

    // show widget tree and fall into gtk event loop
    gtk_widget_show(window);
    gtk_main();

    return 0;
}

